features/currency/managing_currency/browsing_currencies.feature

```diff
@@ -0,0 +1,17 @@
+@managing_currencies
+Feature: Browsing currencies
+    In order to see all currencies in the store
+    As an Administrator
+    I want to browse currencies
+
+    Background:
+        Given the store has currency "Euro"
+        And the store has currency "British Pound Sterling"
+        And the store has currency "US Dollar"
```

Comment:
Wdyt about introducing step ``Given the store has currencies :firstCurrency, :secondCurrency and :thirdCurrency``, as it's done, for example, with taxons?